RealmSwift-swift2.2/LinkingObjects.swift

```diff
@@ -111,7 +111,7 @@ public final class LinkingObjects<T: Object>: LinkingObjectsBase {
         super.init(fromClassName: className, property: propertyName)
     }
 
-    /// Returns a description of the objects contained within the linking objects.
+    /// Returns a description of the objects represented by the linking objects.
```

Comment:
Agreed. Can the objects a `LinkingObjects` links to just be called the “linking objects”?

e.g. “Find the maximum value of the property among the linking objects.”

If that’s too awkward, would “linked objects” work as a generic descriptor instead?

> On Jun 16, 2016, at 11:40 AM, Thomas Goyne <notifications@github.com> wrote:
> 
> In RealmSwift-swift2.2/LinkingObjects.swift <https://github.com/realm/realm-cocoa/pull/3754#discussion_r67401089>:
> 
> > @@ -111,7 +111,7 @@ public final class LinkingObjects<T: Object>: LinkingObjectsBase {
> >          super.init(fromClassName: className, property: propertyName)
> >      }
> >  
> > -    /// Returns a description of the objects contained within the linking objects.
> > +    /// Returns a description of the objects represented by the linking objects.
> As above; the phrase "the objects represented by the linking objects" just doesn't really work.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/realm/realm-cocoa/pull/3754/files/06f4795ac7b9fbbfadcca81a8136bc922d491003#r67401089>, or mute the thread <https://github.com/notifications/unsubscribe/ADhCjjHAbzakxNur2AtSw_XTtJxhDUh6ks5qMZizgaJpZM4I26JH>.
>